src/app/containers/Header/ScriptLink/index.jsx

```diff
@@ -21,10 +22,16 @@ export const getVariantHref = ({ path, params, variant }) =>
 const ScriptLinkContainer = () => {
   const { setPreferredVariantCookie } = useContext(UserContext);
   const { service, script, scriptLink } = useContext(ServiceContext);
+  const { enabled: scriptLinkEnabled } = useToggle('scriptLink');
+  const { enabled: variantCookieEnabled } = useToggle('variantCookie');
 
   const { text, offscreenText, variant } = scriptLink;
   const { path, params } = useRouteMatch();
 
+  if (!scriptLinkEnabled) {
```

Comment:
Could you move this check before the two lines of destructuring above? If we've disabled the script link then we don't need to bother calling the hook above :D

Reply:
Moved this check to be before destructuring `scriptLink `, but `useRouteMatch()` still needs to be placed above because of this rule:

<img width="926" alt="Screenshot 2020-03-09 at 09 20 46" src="https://user-images.githubusercontent.com/17802955/76202813-5fc34e80-61ed-11ea-903c-877a1fc1c3dd.png">